### PR TITLE
fix(api): remove browserType.connect()

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
+# Contributing
+
+## How to Contribute
+
+### Configuring python environment
+
+The project required python version 3.8+. To set it as default in the environment run the following commands:
```
Comment (dgozman) [inline CONTRIBUTING.md:7]: typo: required -> requires

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+Install required dependencies:
+
+```bash
+python -m pip install --upgrade pip wheel
+pip install -r local-requirements.txt
+pip install -e .
+```
+
+For more details look at the [CI configuration](https://github.com/microsoft/playwright-python/blob/master/.github/workflows/ci.yml).
```
Comment (dgozman) [inline CONTRIBUTING.md:22]: I think we can do a relative link here.

**Review by dgozman — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (yury-s) [reply CONTRIBUTING.md:7]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (yury-s) [reply CONTRIBUTING.md:22]: Done.

**Comment by vania-pooh** [comment]
@yury-s what is the expected way of directly connecting to web-socket having `connect()` method removed? Without such feature parallel remote execution is impossible.

**Comment by yury-s** [comment]
We are going to implement connect at the Playwright level, i.e. the client would be able to do something like `new Playwright(<websocket address>` and from that point talk to the remote server same way as if it was running locally. Otherwise with the previous connect API we first need to establish connection to local playwright driver and then call connect on it passing url for each browser individually. This seemed sub-optimal.

**Comment by vania-pooh** [comment]
@yury-s: any related issues or time estimates?

**Comment by yury-s** [comment]
Just filed https://github.com/microsoft/playwright/issues/4687 We'll likely have it soon given that we are deprecating `connect()` but probably after 1.7 (which we is gonna be next week or so).

**Pull request merged**